airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp

```diff
@@ -146,7 +146,27 @@
     }
 
     function createPlayer() {
-        $('#audioPlayer').get(0).addEventListener("ended", onEnded);
+
+        // Handler player events
+        var player = $('#audioPlayer').get(0);
+        player.addEventListener("ended", onEnded);
+        player.addEventListener("canplay", function() { player.play(); });
+
+        // Verbose debug logging for player events
+        player.addEventListener("canplay", function(ev) { console.debug("Received audio event: canplay"); console.debug(ev); })
+        player.addEventListener("loadedmetadata", function(ev) { console.debug("Received audio event: loadedmetadata"); console.debug(ev); })
+        player.addEventListener("seeking", function(ev) { console.debug("Received audio event: seeking"); console.debug(ev); })
+        player.addEventListener("seeked", function(ev) { console.debug("Received audio event: seeked"); console.debug(ev); })
+        player.addEventListener("canplay", function(ev) { console.debug("Received audio event: canplay"); console.debug(ev); })
```

Comment:
Why is there two canplay?

Reply:
Mostly debugging. I intend on removing the extra lines after I'm sure this works.

Reply:
Is line 174 not the same as line 170?

Reply:
You're right, I missed that ;)